CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+## Setting up your working environment
+
+If you want to contribute to the project you'll first need to install the dependencies.
+You can do it via `opam`:
+
+```sh
+$ git clone git@github.com:ocamllabs/dune-release.git
+$ cd dune-release
+$ opam install -t --deps-only .
```

Comment:
We might want to replace that with the proper `opam switch create ./` invocation, what do you think?